pkg/proxy/apis/config/types.go

```diff
@@ -191,23 +191,21 @@ type KubeProxyConfiguration struct {
 }
 
 // ProxyMode represents modes used by the Kubernetes proxy server. Currently, three modes of proxy are available in
-// Linux platform: 'userspace' (older, going to be EOL), 'iptables' (newer, faster), 'ipvs'(newest, better in performance
-// and scalability).
+// Linux platform: 'iptables' (newer, faster), 'ipvs'(newest, better in performance and scalability).
 //
 // Two modes of proxy are available in Windows platform: 'userspace'(older, stable) and 'kernelspace' (newer, faster).
 //
 // In Linux platform, if proxy mode is blank, use the best-available proxy (currently iptables, but may change in the
 // future). If the iptables proxy is selected, regardless of how, but the system's kernel or iptables versions are
-// insufficient, this always falls back to the userspace proxy. IPVS mode will be enabled when proxy mode is set to 'ipvs',
-// and the fall back path is firstly iptables and then userspace.
+// insufficient, will not work. IPVS mode will be enabled when proxy mode is set to 'ipvs' and the fall back path
+// is firstly iptables.
```

Comment:
Ah, I missed this when I removed the fallback stuff. There is no longer any fallback, and there are no systems that don't support iptables. If you specify "iptables", you get iptables. If you specify "ipvs" you either get ipvs or a fatal error.